src/Lang/Cpp/CppDocGen.ts

```diff
@@ -408,40 +446,16 @@ export class CppDocGen implements IDocGen {
 
         this.cfg.File.fileOrder.forEach((element) => {
             switch (element) {
-                case "brief": {
-                    this.insertBrief(lines);
-                    break;
-                }
-                case "empty": {
-                    lines.push("");
-                    break;
-                }
                 case "file": {
                     this.generateFilenameFromTemplate(lines);
                     break;
                 }
-                case "version": {
-                    this.generateVersionTag(lines);
-                    break;
-                }
-                case "author": {
-                    this.generateAuthorTag(lines);
-                    break;
-                }
-                case "date": {
-                    this.generateDateFromTemplate(lines);
-                    break;
-                }
-                case "copyright": {
-                    this.generateCopyrightTag(lines);
-                    break;
-                }
                 case "custom": {
-                    this.generateCustomTag(lines);
+                    this.generateCustomTag(lines,CommentType.file);
                     break;
                 }
                 default: {
-                    break;
+                    this.generateCommonTag(lines, element);
```

Comment:
I would prefer leaving the default case empty. Otherwise typo's may show up with the custom tags which is probably not what de user expects.

Reply:
That's handled in `generateCommonTag()`. There is a default break case there.

Reply:
That's handled in `generateCommonTag()`. There is a default break case there.

Reply:
My bad. I misread `generateCommonTag` as `generateCustomTag`.